definitions/defaults/global.lua

```diff
@@ -93,276 +95,298 @@ return {
     ["<M-x>"] = "ShowBindingList",
     ["<C-m>"] = "TapTempo",
     ['"'] = {"+snapshots", {
-            ["j"] = "RecallNextSnapshot",
-            ["k"] = "RecallPreviousSnapshot",
-            ["D"] = "DeleteAllSnapshots",
-            ["t"] = "ToggleSnapshotsWindow",
-            ["y"] = "CopyCurrentSnapshot",
-            ["p"] = "PasteSnapshot",
-            ["r"] = "RecallCurrentSnapshot",
-            ["#"] = {"+recall #", {
-                        ["1"] = "RecallSnapshot1",
-                        ["2"] = "RecallSnapshot2",
-                        ["3"] = "RecallSnapshot3",
-                        ["4"] = "RecallSnapshot4",
-                        ["5"] = "RecallSnapshot5",
-                        ["6"] = "RecallSnapshot6",
-                        ["7"] = "RecallSnapshot7",
-                        ["8"] = "RecallSnapshot8",
-                        ["9"] = "RecallSnapshot9",
-            }},
+        ["j"] = "RecallNextSnapshot",
+        ["k"] = "RecallPreviousSnapshot",
+        ["D"] = "DeleteAllSnapshots",
+        ["t"] = "ToggleSnapshotsWindow",
+        ["y"] = "CopyCurrentSnapshot",
+        ["p"] = "PasteSnapshot",
+        ["r"] = "RecallCurrentSnapshot",
+        ["#"] = {"+recall #", {
+            ["1"] = "RecallSnapshot1",
+            ["2"] = "RecallSnapshot2",
+            ["3"] = "RecallSnapshot3",
+            ["4"] = "RecallSnapshot4",
+            ["5"] = "RecallSnapshot5",
+            ["6"] = "RecallSnapshot6",
+            ["7"] = "RecallSnapshot7",
+            ["8"] = "RecallSnapshot8",
+            ["9"] = "RecallSnapshot9",
+        }},
     }},
     [","] = {"+options", {
-               ["p"] = "TogglePlaybackPreroll",
```

Comment:
the indentation is meant to be consistent across the repository, so altering indentation levels is not good code quality